dll/cpl/appwiz/createlink.c

```diff
@@ -246,6 +246,7 @@ WelcomeDlgProc(HWND hwndDlg,
             pContext = (PCREATE_LINK_CONTEXT) ppsp->lParam;
             SetWindowLongPtr(hwndDlg, DWLP_USER, (LONG_PTR)pContext);
             PropSheet_SetWizButtons(GetParent(hwndDlg), 0);
+            SHAutoComplete(GetDlgItem(hwndDlg, IDC_SHORTCUT_LOCATION), 0);
```

Comment:
```suggestion
            SHAutoComplete(GetDlgItem(hwndDlg, IDC_SHORTCUT_LOCATION), SHACF_DEFAULT);
```
You might as well use `SHACF_DEFAULT` explicitly (since the flag equates to 0).

Reply:
OK.